docs/src/api.md

```diff
@@ -15,6 +16,7 @@ OSMAgent
 ```
 
 ## Agent/model retrieval and access
+
```

Comment:
Why have you added all these empty lines between headers and docs? they are not necessary, right? everything compiles without them.

Reply:
I forgot about that. I installed a markdown autoformatter recently, and I forgot that it was enabled and formats documents on save. I'll remove these lines